db.py

```diff
@@ -1,10 +1,24 @@
 import psycopg2
-from sqlalchemy import create_engine, text
+from sqlalchemy import create_engine, Column, Integer, String, DateTime, Boolean, select, func, Table
+from sqlalchemy.orm import declarative_base
+from sqlalchemy.orm import Session
 from sqlalchemy.exc import SQLAlchemyError
 from settings import connection_string
 
 engine = create_engine(connection_string)
-schema_version_table = "schema_version"
+Base = declarative_base()
+
+class Todos(Base):
+    __tablename__ = "todos"
+    id = Column(Integer, primary_key=True, index=True)
```

Comment:
Starting from SQLAlchemy 2.0 it`s recommended to use mapped_column function when declaring  da

```
from sqlalchemy.orm import declarative_base, mapped_column, Mapped
Base = declarative_base()

class User(Base):
    __tablename__ = 'users'
    id: Mapped[int] = mapped_column(primary_key=True)
    name: Mapped[str] = mapped_column()
```
**Links**

- [Discussion on StackOverFlow](https://stackoverflow.com/questions/76498857/what-is-the-difference-between-mapped-column-and-column-in-sqlalchemy)
- [SQLAlchemy2.0 documentation](https://docs.sqlalchemy.org/en/20/orm/quickstart.html)